checker/index.html

```diff
@@ -5,8 +5,9 @@
   <meta charset='utf-8'>
   <title>WebIDL 2 Checker</title>
 
-  <script src='../dist/webidl2.js'></script>
-  <script>
+  <script type='module'>
+    import { parse, validate } from '../index.js';
```

Comment:
nit: could we settle on `"` for the file... we've got a mix of single and double quotes.